web_editor_class_selector/static/src/js/backend/html_field.esm.js

```diff
@@ -5,9 +5,9 @@ import {useService} from "@web/core/utils/hooks";
 
 const {onWillStart} = owl;
 
-patch(HtmlField.prototype, "web_editor_class_selector.HtmlField", {
+patch(HtmlField.prototype, {
     setup() {
-        this._super(...arguments);
+        super.setup();
```

Comment:
Maybe passing `...arguments` is a sane default practice...
```suggestion
        super.setup(...arguments);